presto-main/src/main/java/com/facebook/presto/operator/scalar/TDigestFunctions.java

```diff
@@ -73,4 +75,20 @@ public static Block quantilesAtValuesDouble(@SqlType("tdigest(double)") Slice in
         }
         return output.build();
     }
+
+    @ScalarFunction("scale_tdigest")
+    @Description("Scale a t-digest according to a new weight")
+    @SqlType("tdigest(double)")
+    public static Slice scaleTDigestDouble(@SqlType("tdigest(double)") Slice input, @SqlType(StandardTypes.DOUBLE) double scale)
```

Comment:
The other functions in this class are all experimental. We should either make this function experimental, or have a plan to productionize the other functions.

On another point, since all functions are experimental, they're missing documentation.

Reply:
@caithagoras we can probably remove the experimental tag now.

Reply:
Per our offline discussion, I added the `EXPERIMENTAL` tag to this new function (and will push a separate change to take it off all the functions here, and add documentation). I also removed the `private scaleTDigest` method since it isn't very relevant here where we support only one `DOUBLE` type

cc: @tdcmeehan